Modules/Devices/src/Devices.Application/Identities/Queries/GetDeletionProcesses/GetDeletionProcessesResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Backbone.Modules.Devices.Application.DTOs;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+
+namespace Backbone.Modules.Devices.Application.Identities.Queries.GetDeletionProcesses;
+public class GetDeletionProcessesResponse
+{
+    public GetDeletionProcessesResponse(Identity identity)
+    {
+        DeletionProcesses = identity.DeletionProcesses
+            .Select(p => new IdentityDeletionProcessDTO(p))
+            .ToList();
+    }
```

Comment:
I don't think this makes much sense - what if we would have a list of `IdentityDeletionProcess` objects from the get go? In other words, passing an `Identity` is not that great, converting the elements into DTOs is fine.

Reply:
So should I pass the deletion processes as a (read-only) list to the constructor?

```c#
public GetDeletionProcessesResponse(IReadOnlyList<IdentityDeletionProcess> processes)
```

Reply:
I would say so, yes.

Reply:
Yeah, that's probably better when it comes to reusability. Because there may be cases where we don't have an Identity.

Reply:
I added it in the new commit